src/app.controller.ts

```diff
@@ -1,12 +1,17 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Param } from "@nestjs/common";
 import { AppService } from './app.service';
 
-@Controller()
+@Controller('/api')
 export class AppController {
   constructor(private readonly appService: AppService) {}
 
-  @Get()
-  getHello(): string {
-    return this.appService.getHello();
+  @Get('/quiz')
+  getQuiz(): string {
+    return this.appService.getQuiz()
+  }
+
+  @Get('/quiz/:id')
+  getOneQuiz(@Param('id') id : string): string {
+    return this.appService.getOneQuiz(id)
```

Comment:
Please, create a quiz folder with its quiz.controller, service, and module. and move them there